presto-main/src/main/java/io/prestosql/server/PluginClassLoader.java

```diff
@@ -143,21 +139,4 @@ private static String classNameToResource(String className)
     {
         return className.replace('.', '/');
     }
-
-    @SuppressWarnings("JavaReflectionMemberAccess")
-    private static ClassLoader findPlatformClassLoader()
-    {
-        try {
-            // use platform class loader on Java 9
-            Method method = ClassLoader.class.getMethod("getPlatformClassLoader");
```

Comment:
This is a dedicated method on Java 9 to access class loader that you want to use here in Presto.